src/api/app/models/relationship.rb

```diff
@@ -177,7 +177,8 @@ def event_parameters
     parameters = { who: User.session.login,
                    user: user&.login,
                    group: group&.title,
-                   role: role.title }
+                   role: role.title,
+                   notifiable_id: id.to_i }
```

Comment:
You expect this to me something else than an int?

Reply:
No